packages/gateway/tests/integration/connection.spec.ts

```diff
@@ -120,7 +119,7 @@ const createUws = async (options: {
         }
         // retrieve listening port
         uwsToken = token
```

Comment:
```suggestion
```

Reply:
For some reason GitHub doesn't allow me to select three lines and make one suggestion, so please check the other two below